docs/src/model_setup/buoyancy_and_equation_of_state.md

```diff
@@ -157,7 +157,7 @@ SeawaterBuoyancy{Float64}:
 ├── gravitational_acceleration: 1.3
 └── equation of state: LinearEquationOfState(thermal_expansion=0.000167, haline_contraction=0.00078)
 
-julia> model = NonhydrostaticModel(; grid, buoyancy=buoyancy, tracers=(:T, :S))
+julia> model = NonhydrostaticModel(; grid, buoyancy, tracers=(:T, :S))
```

Comment:
Why is `buoyancy=buoyancy` bad? In general I feel like keyword arguments are more user-friendly (with keywords a user doesn't need to know the order of arguments).